src/yum-plugins/tracer_upload.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import json
+import httplib
+import urllib
+import time
+
+from yum.plugins import PluginYumExit, TYPE_CORE, TYPE_INTERACTIVE
+from rhsm.config import RhsmConfigParser, initConfig
+from tracer import Query
```

Comment:
This will fail on below el7, should I wrap it in a try & except or should we not install it on below el7 using the spec file?